live-examples/js-examples/bigint/bigint-tostring.html

```diff
@@ -0,0 +1,11 @@
+<pre>
+<code id="static-js">console.log(256n.toString());
+// expected output: "256"
```

Comment:
It might be more illustrative to use the same bigint value for all three examples?

```js
console.log(1024n.toString());
// expected output: "1024"

console.log(1024n.toString(2));
// expected output: "10000000000"

console.log(1024n.toString(16));
// expected output: "400"
```

...just a suggestion though.